Automattic/newspack-newsletters
# Add new layouts: Daily/Weekly (no image), Breaking News, Breaking News (no image), and Support

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds 4 new layouts:

* Daily/Weekly (no image)
* Breaking News
* Breaking News (no image)
* Support

And "Template 1" has been renamed to "Daily/Weekly".

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Add a new Newsletter
3. Do you see the new layouts?
4. Insert one of the new layouts
5. Does it work as expected?
6. Repeat for the other layouts

### Other information:
